### PR TITLE
Update Agents to latest versions

### PR DESCRIPTION
**Pull request by cirego**
This change is [Reviewable](https://reviewable.io/reviews/flowmill/flowmill-k8s/13)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+## [0.3.0] - 2019-11-22
+### Changed
+    - New Agent version with multiple enhancements
+    - Intake and Services now hosted on flowmill.com
+    - Reduced number of required fields in chart
+    - Automatically create or reuse agent key secret
+    - Add flag to have Agent install kernel headers into its own container
```
Comment (juchem) [inline CHANGELOG.md]: only for GCP COS or Ubuntu hosts

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
+## [0.3.0] - 2019-11-22
```
Comment (juchem) [inline CHANGELOG.md]: shouldn't this match the agent version tag?

Comment (cirego) [reply CHANGELOG.md]: Technically, this is the chart version. Which I should also update...

File: CHANGELOG.md (hunk 1 above)
Comment (cirego) [reply CHANGELOG.md]: I'll remove this note then.

File: CHANGELOG.md (hunk 2 above)
Comment (juchem) [reply CHANGELOG.md]: Wouldn't it make sense to keep it in sync? Not a big deal, though, but it helps reduce confusion

Comment (cirego) [reply CHANGELOG.md]: I did update this to match the agent version.

**Pull request merged**